api/lib/appInsights.ts

```diff
@@ -18,6 +18,7 @@ if (showFeature(FEATURE_APP_INSIGHTS_ENABLED)) {
         .setAutoCollectConsole(true)
         .setUseDiskRetryCaching(true)
         .setSendLiveMetrics(true)
+        .setDistributedTracingMode(applicationinsights.DistributedTracingModes.AI_AND_W3C)
```

Comment:
Why are we doing this as a part of noc?

Reply:
I have no idea as this is the base PR when we followed up NoC, I saw this was added by you @udaydenduluri33 , so we need to remove it?

Reply:
@rayliangatos please remove it

Reply:
anything else apart from this needs to be removed? @udaydenduluri33